.idea/vcs.xml

```diff
@@ -1,27 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-    <component name="CommitMessageInspectionProfile">
-        <profile version="1.0">
-            <inspection_tool class="BodyLimit" enabled="true" level="WARNING" enabled_by_default="true" />
-            <inspection_tool class="SubjectBodySeparation" enabled="true" level="ERROR" enabled_by_default="true" />
-            <inspection_tool class="SubjectLimit" enabled="true" level="WARNING" enabled_by_default="true" />
-        </profile>
-    </component>
-    <component name="IssueNavigationConfiguration">
-        <option name="links">
-            <list>
-                <IssueNavigationLink>
-                    <option name="issueRegexp" value="[A-Z]+\-\d+" />
-                    <option name="linkRegexp" value="https://issues.apache.org/jira/browse/$0" />
-                </IssueNavigationLink>
-                <IssueNavigationLink>
-                    <option name="issueRegexp" value="(apache/calcite)?#(\d+)" />
-                    <option name="linkRegexp" value="https://github.com/apache/calcite-avatica/pull/$2" />
-                </IssueNavigationLink>
-            </list>
-        </option>
-    </component>
-    <component name="VcsDirectoryMappings">
-        <mapping directory="$PROJECT_DIR$" vcs="Git" />
-    </component>
-</project>
+  <component name="CommitMessageInspectionProfile">
```

Comment:
Did you mean to include this @chrisdennis  ?

It doesn't seem related to the depracations.

Reply:
I did not... I guess IntelliJ got over-eager with the reformatting and I did an ill-advised `git add .`.